install/empty_data.php

```diff
@@ -1774,7 +1774,31 @@ public function getEmptyData(): array
                 'itemtype' => 'Plugin',
                 'num' => '8',
                 'rank' => '7',
-            ]
+            ], [
+                'itemtype' => 'Glpi\Event',
+                'num' => '155',
+                'rank' => '1',
+            ], [
+                'itemtype' => 'Glpi\Event',
+                'num' => '156',
+                'rank' => '2',
+            ], [
+                'itemtype' => 'Glpi\Event',
+                'num' => '157',
+                'rank' => '3',
+            ], [
+                'itemtype' => 'Glpi\Event',
+                'num' => '158',
+                'rank' => '4',
+            ], [
+                'itemtype' => 'Glpi\Event',
+                'num' => '159',
+                'rank' => '5',
+            ], [
+                'itemtype' => 'Glpi\Event',
+                'num' => '160',
+                'rank' => '6',
+            ],
```

Comment:
I am not sure this default order is the best, but I am not sure either that I have a better proposal.

Reply:
Indeed.